src/components/ColorPicker/BrandColorPicker.tsx

```diff
@@ -0,0 +1,101 @@
+import { Slider } from "@components/Slider/Slider";
+import IconCheck from "@elements/Icon/Generated/IconCheck";
+import IconImageGrid2 from "@elements/Icon/Generated/IconImageGrid2";
+import IconSearch from "@elements/Icon/Generated/IconSearch";
+import IconUnorderedList from "@elements/Icon/Generated/IconUnorderedList";
+import { IconSize } from "@elements/Icon/IconSize";
+import { TextInput } from "@elements/TextInput/TextInput";
+import { toColor } from "@utilities/colors";
+import { debounce } from "@utilities/debounce";
+import { merge } from "@utilities/merge";
+import React, { FC, useEffect, useState } from "react";
+import { ColorPickerProps } from "./ColorPicker";
+
+const find = (haystack?: string, needle = "") =>
+    haystack?.toLocaleLowerCase().includes(needle.toLocaleLowerCase().trim());
+
+enum BrandColorView {
+    List = "List",
+    Grid = "Grid",
+}
+
+export const BrandColorPicker: FC<ColorPickerProps> = ({ palettes: defaultPalettes = [], currentColor, onSelect }) => {
+    const views = [
+        { id: BrandColorView.List, icon: <IconUnorderedList />, ariaLabel: "List" },
+        { id: BrandColorView.Grid, icon: <IconImageGrid2 />, ariaLabel: "Grid" },
+    ];
+    const [view, setView] = useState(views[0].id);
+    const [query, setQuery] = useState("");
+    const [palettes, setPalettes] = useState(defaultPalettes);
+    useEffect(() => {
+        setPalettes(
+            defaultPalettes
+                .filter(({ title, colors }) => find(title, query) || colors.some(({ name }) => find(name, query)))
+                .map(({ title, colors }) => ({
+                    title,
+                    colors: colors.some(({ name }) => find(name, query))
+                        ? colors.filter(({ name }) => find(name, query))
+                        : colors,
+                })),
+        );
+    }, [query]);
+
+    return (
+        <div className="tw-flex tw-flex-col tw-gap-5">
+            <div className="tw-flex tw-gap-3">
+                <div className="tw-flex-1">
+                    <TextInput
+                        decorator={<IconSearch />}
+                        placeholder="Search"
+                        onInput={debounce((value) => setQuery(value), 200)}
+                    />
+                </div>
+                <div>
+                    <Slider
```

Comment:
The slider is bigger than the search input, probably due to too much padding.
There are several changes for the Slider already, so I'd suggest to fix the height when taking care of [this ticket](https://app.clickup.com/t/nkfgmx).

Reply:
I fixed it on `main`, maybe merge it back to this branch to fix it?